kube_controller_manager/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - Kube_controller_manager
 
+## 1.8.0 / 2021-03-07
+
+* [Added] Add support for Kubernetes leader election based on Lease objects. See [#8535](https://github.com/DataDog/integrations-core/pull/8535).
+* [Fixed] Bump base package requirement. See [#8770](https://github.com/DataDog/integrations-core/pull/8770).
+* [Fixed] Bump minimum base package version. See [#8443](https://github.com/DataDog/integrations-core/pull/8443).
```

Comment:
Those two PRs are changing the same thing: the `datadog-checks-base` minimum version. As the later PR overrides the change of the first one, it may be useless to have two lines in the changelog.

```suggestion
* [Fixed] Bump minimum base package version. See [#8770](https://github.com/DataDog/integrations-core/pull/8770) and [#8443](https://github.com/DataDog/integrations-core/pull/8443).
```